assets/src/edit-story/components/library/panes/shapes/shapesPane.js

```diff
@@ -55,8 +55,8 @@ function ShapesPane(props) {
       )}
       <Section title={__('Basic shapes', 'web-stories')}>
         <SectionContent>
-          {MASKS.map((mask) => (
-            <ShapePreview mask={mask} key={mask.type} />
+          {MASKS.filter((mask) => mask.showInLibrary).map((mask) => (
```

Comment:
@dvoytenko Okay, now only masks with `showInLibrary` as `true` will appear in the shapes panel.  This is how the shapes panel looks now (all the new ones are hidden):
![image](https://user-images.githubusercontent.com/40646372/86065583-cbdbf380-ba24-11ea-8d16-116ed20600b5.png)


Does this work?

Reply:
Yep. Approved.